include/swift/RemoteInspection/TypeRefBuilder.h

```diff
@@ -476,8 +476,8 @@ class TypeRefBuilder {
       return ReflectionInfos;
     }
 
-    /// Load unsubstituted field types for a nominal type.
-    RemoteRef<FieldDescriptor> getFieldTypeInfo(const TypeRef *TR);
+    std::unique_ptr<FieldDescriptorBase>
+    getFieldDescriptor(const TypeRef *TR) override;
```

Comment:
where did the comment go?

Reply:
I moved the whole function, including comment, to a private section.